third-party/bazel/edm-wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+EDM_LINK=git+https://github.com/Everest/everest-dev-environment.git#subdirectory=dependency_manager
+mkdir -p venv > /dev/null
+TARGET_PATH=$(realpath venv)
+pip3 install --upgrade -t $TARGET_PATH $EDM_LINK > /dev/null
+export PYTHONPATH=$TARGET_PATH
```

Comment:
should we restore the old PYTHONPATH after the command?

Reply:
We exit after the command, why bother restoring anything?

Reply:
ah brainfart - we never source this but just run